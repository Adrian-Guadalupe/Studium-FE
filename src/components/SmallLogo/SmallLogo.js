import React from 'react';
import styled from 'styled-components';
import logo from '../../images/smalllogo.png';

/**
 * Small Logo
 * @component
 * @example
 * return (
 *  <SmallLogo />
 *  )
 */
export const SmallLogo = () => {
  return (
    <StyledSmallLogo>
      <img src={ logo } alt="Logo"/>
    </StyledSmallLogo>
  );
};

const StyledSmallLogo = styled.div`
  margin: 121px auto 0 auto;
`;
