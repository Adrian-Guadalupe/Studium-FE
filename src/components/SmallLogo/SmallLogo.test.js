import React from 'react';
import { render } from '@testing-library/react';
import {SmallLogo} from './SmallLogo.js';
import renderer from 'react-test-renderer';



// check logo in alt
test('check logo is in alt', () => {
  const { getByAltText } = render(<SmallLogo />);
  const linkElement = getByAltText(/logo/i);
  expect(linkElement).toBeInTheDocument();
  // expect(render(<App />)).toMatchSnapShot()
});

test('SmallLogo render correctly', () => {
  const tree = renderer.create(<SmallLogo />).toJSON();
  expect(tree).toMatchSnapshot();
});


