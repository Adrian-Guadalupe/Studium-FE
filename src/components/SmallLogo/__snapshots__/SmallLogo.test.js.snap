// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallLogo render correctly 1`] = `
.c0 {
  margin: 121px auto 0 auto;
}

<div
  className="c0"
>
  <img
    alt="Logo"
    src="smalllogo.png"
  />
</div>
`;
